config/labelbot.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
+
+    /*
+    | K for KNN
+    */
+    'K' => 100,
+
+    /*
+    | N for top N labels
+    */
+    'N' => 3,
```

Comment:
Please elaborate these comments so you still understand what the variables mean if you come back in 5 years :wink: